connman/vpn/vpn-provider.c

```diff
@@ -1302,12 +1312,15 @@ static void reset_error_counters(struct vpn_provider *provider)
 	if (!provider)
 		return;
 
+	DBG("provider %p", provider);
+
 	provider->auth_error_counter = provider->conn_error_counter = 0;
 }
 
 static int vpn_provider_save(struct vpn_provider *provider)
 {
 	GKeyFile *keyfile;
+	const char* value;
```

Comment:
s/* / */

Reply:
Oops, fixed in 3c87042b46fd46316de0631d898c0b0c57bf8c8e